packages/common/src/hardhat/fixtures/default.js

```diff
@@ -0,0 +1,67 @@
+const { interfaceName } = require("../../Constants");
+const { RegistryRolesEnum } = require("../../Enums");
+
+async function runDefaultFixture({ deployments }) {
+  const setup = deployments.createFixture(async ({ deployments, getNamedAccounts, web3 }) => {
+    const { padRight, toWei, utf8ToHex } = web3.utils;
+    await deployments.fixture();
+    const { deployer } = await getNamedAccounts();
+
+    const getDeployment = async (name) => {
+      const contract = await deployments.get(name);
+      new web3.eth.Contract(contract.abi, contract.address);
+    };
+
+    // Setup finder.
+    const finder = await getDeployment("Finder");
+
+    const addToFinder = async (deploymentName, finderName) => {
+      const { address } = await deployments.get(deploymentName);
+      const hexName = padRight(utf8ToHex(finderName));
+      await finder.methods.changeImplementationAddress(hexName, address).send({ from: deployer });
+    };
+
+    await addToFinder("FinancialContractsAdmin", interfaceName.FinancialContractsAdmin);
+    await addToFinder("Voting", interfaceName.Oracle);
+    await addToFinder("Registry", interfaceName.Registry);
+    await addToFinder("Store", interfaceName.Store);
+    await addToFinder("IdentifierWhitelist", interfaceName.IdentifierWhitelist);
+    await addToFinder("AddressWhitelist", interfaceName.CollateralWhitelist);
+    await addToFinder("OptimisticOracle", interfaceName.OptimisticOracle);
+    await addToFinder("Bridge", interfaceName.Bridge);
+    await addToFinder("GenericHandler", interfaceName.GenericHandler);
```

Comment:
Also need `FundingRateStore` right? Or are we not yet adding the Perp-related contracts to prod set up

Reply:
I don't think we have a funding rate store. I think that mapping element is a leftover from before we deleted that contract.

Reply:
I can remove the mapping element, though!

Reply:
Done.

Reply:
Oh you're right. Can you remove the mapping in this PR

Reply:
Just did it :)